### ISSUE TITLE
Error when running examples/list_formats.lua

### ISSUE THREAD

**Issue by amsitlab** [stetre/moonsndfile#1]
```
┌[ ~/github.com/stetre/moonsndfile ]─[ ⑂master* ]
└─> lua5.3 examples/list_formats.lua
lua5.3: examples/list_formats.lua:14: unexpected error (enums.c, 201)
stack traceback:
    [C]: in function 'moonsndfile.get_format_list'                                                   
        examples/list_formats.lua:14: in main chunk         
   [C]: in ?```

**Comment by stetre** [comment]
Looks like libsndfile reports an unknown format. May I ask you which version of libsndfile you are using?
(Just run examples/version.lua and copy here the output).

**Comment by amsitlab** [comment]
> Looks like libsndfile reports an unknown format. May I ask you which version of libsndfile you are using? (Just run examples/version.lua and copy here the output).

```
┌[ ~ ]─[ ]
└─> apt list --installed | grep libsndfile

WARNING: apt does not have a stable CLI interface. Use with caution in scripts.

libsndfile/stable,now 1.0.31 aarch64 [installed,automatic]
```

**Comment by amsitlab** [comment]
> Looks like libsndfile reports an unknown format. May I ask you which version of libsndfile you are using? (Just run examples/version.lua and copy here the output).

```
┌[ ~/github.com/stetre/moonsndfile ]─[ ⑂master* ]
└─ lua5.3 examples/version.lua
MoonSndFile version: MoonSndFile 0.1
libsndfile.so version: libsndfile-1.0.31
libsndfile.so version major: 1
libsndfile.so version minor: 0
libsndfile.so version patch: 31
```

**Comment by stetre** [comment]
Thank you. Most likely the problem is due to MoonSndfile being aligned to libsndfile-1.0.28. Newer formats were added in later versions (which I didn't notice were out, my bad). I'll update it as soon as possible.

**Comment by stetre** [comment]
The latest commit is aligned to libsndfile-1.0.31 and should solve the problem. Please let me know if it does. Thank you

**Comment by opnoobkiller** [comment]
I have an issue with the moonfile module not being found at all, I'm on windows 10 I could only run the git command, not sudo for linux

**Comment by stetre** [comment]
Did you follow the [instructions](https://github.com/stetre/moonlibs#installing-on-windows)?

**Comment by opnoobkiller** [comment]
Yeah I ended up following the instrctions, got everything to install using
$, but then when I tried to require it in my project on one of my desktop
folders it said moonsndfile is not found, I think it's refering to
moonsndfile.lua

**Comment by stetre** [comment]
It's referring to the shared object, aka 'dll' in Windows' jargon. After installation it should be in the folder `C:\msys32\mingw64\bin` (or `C:\msys32\mingw32\bin` if you have a 32-bit system), and named `moonsndfile.dll`.

Have you set the PATH variable properly?
